kickoff/jsonexport.py

```diff
@@ -180,12 +180,20 @@ def generateLocalizedBuilds(buildsVersionLocales, l10nchangesets, lastVersion):
     return buildsVersionLocales
 
 
-def fillAuroraVersion(buildsVersionLocales):
-    for localeCode in config.SUPPORTED_AURORA_LOCALES:
+def fillPrereleaseVersion(buildsVersionLocales, channel='aurora'):
+    # Our default values are for Aurora
+    locales = config.SUPPORTED_AURORA_LOCALES
```

Comment:
In Python, we usually do that this way:
` locales = config.SUPPORTED_NIGHTLY_LOCALES if channel == 'nightly' else config.SUPPORTED_AURORA_LOCALES`
(same for version)